app/experimenter/experiments/bugzilla.py

```diff
@@ -155,6 +158,18 @@ def format_creation_bug_body(experiment, extra_fields):
     return bug_data
 
 
+def format_summary(experiment):
+    experiment_string = str(experiment)
+
+    if len(experiment_string) > EXPERIMENT_NAME_MAX_LEN:
```

Comment:
Do you need this check, can't you just always truncate and if it's less then it's basically a no-op?